app/models/user.rb

```diff
@@ -647,8 +646,97 @@ def banned_at_date?(date)
     groups
   end
 
+  def self.panel_list
+    panel_pages = PanelController.panel_pages
```

Comment:
Rails has a whole host of methods related to converting strings between camel case / kebap case / snake case / underscore case etc.

I feel like the only purpose of `PanelController.panel_pages` is to store the association between some string keys and the same strings in kebap-case format again as values. So maybe this PR would be a good opportunity to clean that up?

Reply:
Actually not just that. panel_pages is used in many places. The panel_pages is the source of truth for the list of panel pages. This list is used in PanelPages.jsx to map it to the title and the react component/hyperlink. In this case, panel_pages is used to refer to the panel pages in ease. Hope this clarifies.

Reply:
I've made the changes as you suggested.